zerver/lib/users.py

```diff
@@ -19,6 +19,7 @@
 
 from zulip_bots.custom_exceptions import ConfigValidationError
 
+import re
```

Comment:
This import should go before `unicodedata`; there's an ordering that imports go in.